app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,74 @@
+class PostsController < ApplicationController
```

Comment:
Remember to remove this controller, migrations and view as marked in the other pr, since you aren't going to use the post model